InternetSemLimites/api/urls.py

```diff
@@ -12,4 +12,5 @@
     url(r'^(?P<abbr>[\w]{2})/shame/$', shame_by_state, name='shame_by_state'),
     url(r'^provider/new/$', provider_new, name='new'),
     url(r'^provider/(?P<pk>[\d]+)/$', provider_detail, name='provider'),
+    url(r'^provider/(?P<pk>[\d]+)/edit/$', provider_edit, name='edit'),
```

Comment:
Sobre essa questão, Django não suporta `PUT` por padrão, somente `POST`.

Poderíamos usar o `POST` mesmo ou usar o Django REST Framework. Porém, acho o último meio overkill se for usar em somente uma view.

Fiz com o `POST` e funcionou bem, mas não ficar tão RESTful 😞